classes/controller/FrontController.php

```diff
@@ -1465,7 +1465,7 @@ public function getTemplateVarUrls()
             }
 
             foreach ($assign_array as $assign_key => $assign_value) {
-                if (substr($assign_value, 0, 1) == '/' || $this->ssl) {
+                if ($this->ssl) {
```

Comment:
I'm not sure to understand why you removed it here 🤔

Reply:
Check the comment that I give to @kpodemski : https://github.com/PrestaShop/PrestaShop/pull/29087#discussion_r973119165. (I close this one).